src/pages/DetailPage/Components/Review.js

```diff
@@ -0,0 +1,93 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import { FaRegThumbsUp } from 'react-icons/fa';
+
+export default function Review() {
+  const [review, setReview] = useState([]);
+
+  useEffect(() => {
+    fetch('data/ReviewData.json')
+      .then(res => res.json())
+      .then(res => setReview(res));
+  }, []);
+
+  return review.map((data, idx) => (
```

Comment:
```suggestion
  const [reviews, setReviews] = useState([]);

  useEffect(() => {
    fetch('data/ReviewData.json')
      .then(res => res.json())
      .then(res => setReview(res));
  }, []);

  return reviews.map((review, idx) => (
```
data는 안 좋은 변수명인거 같은데 이렇게 지어보시는게 어떨까요?

Reply:
넵 조금 더 가독성 좋은 변수명으로 수정하였습니다.